File: VJudge DAY 01 ( JAVA ) BRACU ACM Student Chapter/T.java
import java.util.Scanner;
public class T {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    int count = input.nextInt();
    for(int i=1; i<=count; i++){
      int a = input.nextInt();
      int b = input.nextInt();
      int c = input.nextInt();
      System.out.println((a+b+c==180) ? "YES" : "NO");
    }
  }
}

File: VJudge DAY 01 ( C++ )  BRACU ACM Student Chapter/T.c
#include<stdio.h>
int main()
{
    int a, b, c, i, count;
    scanf("%d", &count);
    for(i=1; i<=count; i++)
    {
        int sum=0;
        scanf("%d %d %d", &a, &b, &c);
        sum = a+b+c;
        sum==180 ? printf("YES\n") : printf("NO\n");
    }
    return 0;
}
